README.md

```diff
@@ -30,6 +30,21 @@ e.g. on a Linux system you might want to build for local use like this:
 
 This will put an IrrlichtMtTargets.cmake file into the cmake directory in the current build directory, and it can then be imported from another project by pointing `find_package()` to the build directory, or by setting the `CMAKE_PREFIX_PATH` variable to that same path.
 
+on Windows system:
+
+It is highly recommended to use vcpkg as package manager.
+
+After you successfully built vcpkg you can easily install the required libraries:
+
+	vcpkg install zlib zstd curl[winssl] openal-soft libvorbis libogg libjpeg-turbo sqlite3 freetype luajit gmp jsoncpp opengl-registry --triplet x64-windows
+	
+Run the following script in PowerShell:
+
+	git clone https://github.com/minetest/irrlicht
+	cd irrlicht
+	cmake . -G"Visual Studio 17 2022" -A"Win64" -DCMAKE_TOOLCHAIN_FILE=D:/vcpkg/scripts/buildsystems/vcpkg.cmake -DBUILD_SHARED_LIBS=OFF
+	cmake --build .
```

Comment:
`--config Release`?

Reply:
Also might be good to specify a build directory (`-B build`), because it's a lot cleaner to manage. The README is an example people may learn from, so it should recommend good practices for building the project.